src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceGeneral2.java

```diff
@@ -63,4 +63,10 @@ void method() throws Exception {
             a.equals("");
         }
     }
+
+    void issue11973() {
+        int i = -1;
+        switch (i) {
+        }
+    }
```

Comment:
@stoyanK7 please explain why we cannot add this example to new input file.

Reply:
To **a** new input file or to **the** new input file?

Reply:
https://github.com/checkstyle/checkstyle/pull/12052/files#diff-70dea49acafeb398ee0ccd4d208360b12c5c169feafa2f3465f77d15874533c8

Reply:
Because we already have it here? I mean if you want to repeat it in each test, sure. Done.

Reply:
Do you want it moved to the new file or just added as well?

Reply:
No particular reason. It doesn't matter at the end of the day, does it? Moved it over to the new input file. Done

Reply:
> It doesn't matter at the end of the day, does it?

Please re-read https://github.com/checkstyle/checkstyle/pull/12052#discussion_r967865041 and try to understand viewpoint of reviewer; new test inputs only means that we know that we did not introduce any change in coverage of some condition that we do not immediately recognize. Keep in mind that we run our regression tests over our own code base, so even though your check input is written for one check, we run all checks over all inputs. Yes, your change was minimal, and probably affected nothing, but then we have some grey area. How do we define a minimal change in test input?

It is easier to just follow 1 issue == 1 PR == one (or more) new tests == corresponding number of test inputs. This allows us to focus on the most critical changes (the change in the check itself).